hack/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2021 The Kubernetes-CSI-Addons Authors.
+Copyright 2022 The Kubernetes-CSI-Addons Authors.
```

Comment:
this file was added on Dec 15 2021 why year update is required?

Reply:
Similar concern @Rakshith-R

Reply:
This file is used as boilerplate for creating new `.go` files, updating here will cause the files to have `2022` instead of old `2021`.
@Madhu-1 @Yuggupta27

Reply:
we can easily lose track of updating this file. this should be automated in Makefile where we are generating the files?

Reply:
on another hand, it seems to be modifying already existing file 
```
--- a/api/v1alpha1/zz_generated.deepcopy.go
+++ b/api/v1alpha1/zz_generated.deepcopy.go
@@ -2,7 +2,7 @@
 // +build !ignore_autogenerated
 
 /*
-Copyright 2021 The Kubernetes-CSI-Addons Authors.
+Copyright 2022 The Kubernetes-CSI-Addons Authors.
```

This pr can be put off till later. 

have to manually change year in newly generated files for now.

Reply:
Yes, some files like `api/v1alpha1/zz_generated.deepcopy.go` get regenerated every time. The build checks for changes, so that nobody modifies these generated files.

It is fine to commit these generated files with the change in the boilerplate, just mention that you re-generated them in the commit message.